modules/aws/master/main.tf

```diff
@@ -117,15 +117,9 @@ resource "aws_instance" "master" {
   ), var.extra_tags)}"
 }
 
-resource "aws_elb_attachment" "masters_tnc" {
+resource "aws_elb_attachment" "masters_ctrlp" {
```

Comment:
just `masters_internal` or `ctrlp_internal` ?

Reply:
`ctrlp_internal` between those, long-term it might be possible to consolidate internal and external

Reply:
long-term, @crawford wants to drop external lbs... :P

Reply:
lets do `ctrp_internal`